src/Common/test-runtime/project.json

```diff
@@ -18,6 +18,7 @@
     "Microsoft.DotNet.BuildTools.TestSuite": "1.0.0-prerelease-00520-02",
     "System.Collections.NonGeneric": "4.0.1",
     "System.Collections.Specialized": "4.0.1",
+    "System.Net.WebSockets.Client": "4.0.1-beta-24322-03",
```

Comment:
Agreed to merge this first. 

@mconnew do you like to have an issue opened to track to remove this dependency from test-runtime later or will you just remember it? 😄